sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -7381,6 +7381,31 @@ public async Task CanGetParentContainerClient_WithContainerSAS()
             Assert.IsNotNull(blobItems);
         }
 
+        [RecordedTest]
```

Comment:
For organizational purposes, could you move these tests to `BlobUriBuilderTests.cs` along with the other related tests from [this PR](https://github.com/Azure/azure-sdk-for-net/pull/30271)? Will be good to go once that is handled.

Reply:
Sure, I have updated the PR